flink-kubernetes/src/main/java/org/apache/flink/kubernetes/utils/KubernetesUtils.java

```diff
@@ -122,19 +123,35 @@ public static String getDeploymentName(String clusterId) {
 	 * Get resource requirements from memory and cpu.
 	 *
 	 * @param mem Memory in mb.
-	 * @param cpu cpu.
+	 * @param cpuRequest cpu request.
```

Comment:
Hi Canbin, thanks for your contribution. Not meant to be a grammar police, however, I have seen multiple 'cpu' occurrences in the description and error messages. For standardization, could you please capitalize it as "CPU"? 

Also please pay attention to its plural form, like "number of cpu", as it should be "number of CPUs" in the description text.

Reply:
> For standardization, could you please capitalize it as "CPU"?
Thanks for pointing out this, the new commit fix it.

> Also please pay attention to its plural form, like "number of cpu", as it should be "number of CPUs" in the description text.
They are not introduced by this PR and indeed are not in the scope.